packages/aa/src/index.js

```diff
@@ -74,22 +98,16 @@ async function loadCanonicalNameMap({ rootDir, includeDevDeps, resolve }) {
 }
 
 /**
- * @param {Resolver} resolve
- * @param {string} depName
- * @param {string} packageDir
+ * Resolves `package.json` of a dependency relative to `basedir`
+ *
+ * @param {Resolver} resolve - Resolver function
+ * @param {string} depName - Dependency name
+ * @param {string} basedir - Dir to resolve from
  * @returns {string | undefined}
  */
-function wrappedResolveSync(resolve, depName, packageDir) {
+function wrappedResolveSync(resolve, depName, basedir) {
   const depRelativePackageJsonPath = path.join(depName, 'package.json')
-  try {
-    return resolve.sync(depRelativePackageJsonPath, { basedir: packageDir })
-  } catch (err) {
-    if (!(/** @type {Error} */ (err).message.includes('Cannot find module'))) {
```

Comment:
I think we were silencing the module not found errors to handle that more gracefully elsewhere.